docs/docs/integrations/datadog.mdx

```diff
@@ -1,100 +1,65 @@
-# Integrate DataDog with GrowthBook Feature Flag Data
+# DataDog Integration
 
-By harnessing the power of DataDog's monitoring and alerting capabilities, along with GrowthBook&apos;s powerful REST API, you can create a dynamic system that automatically toggles GrowthBook Feature Flags when error thresholds surpass predefined thresholds.
+## DataDog RUM with Feature Flag Data
 
-## Setup Instructions
+Integrating DataDog&aps;s Real User Monitoring (RUM) with GrowthBook&aps;s feature flag data lets you quickly identify which experience your users were exposed to and if the changes introduced affect the user experience.
 
-### 1. Instrument DataDog with Feature Flag Data
+This guide will cover the initial setup and an advanced example of how to leverage DataDog and GrowthBook&aps;s powerful APIs to create a dynamic system that automatically toggles GrowthBook feature flags based on error thresholds.
 
-DataDog has a number of products that can be used to monitor applications. For this example, we'll focus on how to integrate
-GrowthBook with [DataDog RUM](https://www.datadoghq.com/product/real-user-monitoring/).
+### Setup
 
-To get started, you'll first need to instrument DataDog's RUM with their beta [Feature Flag Tracking](https://docs.datadoghq.com/real_user_monitoring/feature_flag_tracking/) feature.
+:::tip Info
 
-When initializing `datadogRum` you can add `enableExperimentalFeatures` and pass in `feature_flags` to get access to the beta feature.
+If your DataDog RUM browser SDK version is below v5.17.0, [enable feature flags during RUM SDK initialization](https://docs.datadoghq.com/real_user_monitoring/guide/setup-feature-flag-data-collection/?tab=browser#setup).
 
-```ts
-datadogRum.init({
-  applicationId: "sample-application-id",
-  clientToken: "sample-client-token",
-  site: "datadoghq.com",
-  service: "demo-app",
-  env: "dev",
-  version: "1.0.0",
-  sessionSampleRate: 100,
-  sessionReplaySampleRate: 100,
-  trackUserInteractions: true,
-  trackResources: true,
-  trackLongTasks: true,
-  defaultPrivacyLevel: "mask-user-input",
-// This enables the beta feature flag tracking
-  enableExperimentalFeatures: ["feature_flags"],
-});
-
-datadogRum.startSessionReplayRecording();
-```
-
-Then, you can use the `addFeatureFlagEvaluation` method built in to the `datadogRum` package when evaluating GrowthBook Feature Flags.
-
-```ts
-import { datadogRum } from "@datadog/browser-rum";
-import { useFeatureIsOn } from "@growthbook/growthbook-react";
+:::
 
-const welcomeMessage = useFeatureIsOn("welcome-message");
+We can report all feature flag evaluations to DataDog using GrowthBook’s SDK `trackingCallback` function, as shown in the snippet below.
 
-datadogRum.addFeatureFlagEvaluation("welcome-message", welcomeMessage);
+```
+trackingCallback: (experiment, result) => {
```

Comment:
I think this is fine, but we also have a `onFeatureUsage` cb, which seems perfect for this use case. See https://docs.growthbook.io/lib/react#feature-usage-callback